app/js/views/services/default-service.jsx

```diff
@@ -0,0 +1,86 @@
+/* global React */
+
+import UserLogoutButton from 'js/views/user-logout-button';
+import FooterMenu from 'js/views/footer-menu';
+import TagList from 'js/views/tag-list';
+
+export default class Service extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: props.service.tags,
+      tags: []
+    };
+
+    this.foxbox = props.foxbox;
+    this.service = props.service;
+
+    this.onServiceStateChanged = this.onServiceStateChanged.bind(this);
+  }
+
+  componentDidMount() {
+    this.populateTags();
+
+    this.foxbox.addEventListener(
```

Comment:
Yes, you're right. `onServiceStateChanged` must check that the ID of the updated service is the same as the current service one. Let's fix it later.